source/docs/articles/wordpress/importing-a-wordpress-site.md

```diff
@@ -6,53 +6,122 @@ category:
+## Export WordPress via Plugins
+Popular plugins like [Duplicator](/docs/articles/wordpress/clone-a-wordpress-site-with-duplicator-plugin/) and [BackupBuddy](http://ithemes.com/codex/page/BackupBuddy) allow you to create single file archives quickly. For large imports, pasting in a web-readable url (e.g. Dropbox secret link) will be much faster than manually uploading. Make sure a link goes directly to the file, not a landing page.
 
 <div class="alert alert-info" role="alert"> <strong>Note:</strong> Dropbox URL's need to be modified so they end in <code>dl=1</code> instead of the default <code>dl=0</code>. This forces a download of your archive and avoids the Dropbox landing page.</div>  
-
-
-Of course you can make an import archive by hand. You simply dump your existing database into a file ending with “.sql”, and make an archive file (zip or tar.gz) that contains that dump along with all your WordPress code.
-
-## Add and Name a Site
-
-After you have created an account, you can log in and will be directed to your Dashboard.
-![Your sites & account dashboard](/source/docs/assets/images/create-site-dashboard.png)
-Click **Add a site**. You will first be prompted to name the site. The only valid characters are letters, numbers, and dashes. Enter a name and click **Continue**.
-![](/source/docs/assets/images/desk_images/247523.png)
-## Wait While It's Configured
-
-You will then have a short wait while Pantheon creates and allocates the resources for your site's environments. This takes only a few minutes under normal circumstances.
-![](/source/docs/assets/images/desk_images/247524.png)
-## Choose a Start State
+You can simply upload the archive file produced and Pantheon will do the rest.
+<div class="alert alert-danger" role="alert">
+<strong>Warning</strong>:  Archives cannot contain multiple <code>.sql</code> files, otherwise the import will fail.</div>
+
+
+## Manually Create Separate Site Archives
+
+Your codebase is required to import your site into Pantheon, as it will be used to create the initial code repository. This archive should include your entire WordPress codebase, including plugins, themes, configuration files etc.
+
+The code archive must include the following files and directories:
+```nohighlight
+├── index.php
+├── wp-activate.php
+├── wp-config.php
+├── wp-comments-post.php
+├── wp-blog-header.php
+├── wp-admin
+├── wp-cron.php
+├── wp-load.php
+├── wp-links-opml.php
+├── wp-includes
+├── xmlrpc.php
+├── wp-trackback.php
+├── wp-signup.php
+├── wp-settings.php
+├── wp-mail.php
+├── wp-login.php
+├── wp-content
+    ├── index.php
+    ├── mu-plugins
+    ├── themes
+    ├── plugins
+```
+### Create Code Archive
+The code archive should not include the `wp-content/uploads` or any other static assets that should not be tracked in version control. If your codebase contains static files they should be moved to the `wp-content/uploads` directory before export.

```

Comment:
passive >> active? 
"The code archive should not..."
to
"Do not include the `wp-content/uploads` directory or any other static assets that shouldn't be tracked in Git version control...."